### DISCUSSION TITLE
Multiple Schema/Owner in the Database

### DISCUSSION THREAD

**Discussion by GustavoRochaSantos** [prisma/prisma#9342] [Q&A] [answered]
Hello Coders!

In SQL Server, I can create a database and in this database, I can create any schemas/owners I want, with yours tables, like attached.

How can I do this in Prisma? I tried to use Introspect, but just work of dbo schema.

![Screen Shot 2021-09-20 at 2 19 35 PM](https://user-images.githubusercontent.com/12894615/134047849-f3c8652e-a81a-4dc7-8987-c61d5244c898.png)

**Comment by janpio** [comment, marked as answer]
Unfortunately you have to decide between schemas for now until https://github.com/prisma/prisma/issues/1122 is implemented.

For SQL Server there seems to be a workaround documented here: https://github.com/prisma/prisma/issues/1122#issuecomment-825774657

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
